service/src/main/resources/api/service_openapi.yaml

```diff
@@ -558,6 +560,8 @@ components:
         createdBy:
           type: string
           description: Identity of the user who originally created the profile
+        policies:
```

Comment:
🤔 I don't think there even is an implemented update endpoint. I'd exclude this for now to avoid any confusion that updating the TPS policies is even an option.

Reply:
I think the github minimizing was confusing here, this `policies` field is for the `ProfileModel` which is the standard profile response. It's not for `UpdateProfileRequest` like it looks like. Let me know if that's not what happened and you meant something else!

Reply:
Ah! Thanks.